SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -104,6 +104,7 @@ class PositioningViewController: SitumViewController, GMSMapViewDelegate, UITabl
         backButton.title = NSLocalizedString("positioning.back",
             bundle: SitumMapsLibrary.bundle,
             comment: "Button to go back when the user is in the positioning controller (where the map is shown)")
+        customizeBackButtonColor()
```

Comment:
Vexo que este método se chama máis abaixo en `initializeUIElments` que é cando o tema do usuario está cargado. Cando se carga wayfinding aparece o view controller antes de este método?

Tiveche en conta que en este punto non tes o tema do usuario (al menos na primeira execución) que color se establecería aquí?